include/deal.II/lac/dynamic_sparsity_pattern.h

```diff
@@ -615,6 +615,24 @@ class DynamicSparsityPattern : public Subscriptor
   const IndexSet &
   row_index_set() const;
 
+  /**
+   * Return the IndexSet that contains entries for all columns in which at least
+   * one element exists in this sparsity pattern.
+   *
+   * @note in a parallel context, this only considers the locally stored rows.
```

Comment:
This will be rendered funny. Use this:
```suggestion
   * @note In a parallel context, this only considers the locally stored rows.
```